.github/workflows/deploy.yaml

```diff
@@ -396,7 +396,7 @@ jobs:
             participants add \
             --ids-url "http://${{ env.EDC_HOST }}:8282"
         env:
-          REGISTRATION_SERVICE_API_URL: ${{ needs.Deploy-Dataspace.outputs.registration_service_url }}/authority
+          REGISTRATION_SERVICE_API_URL: ${{ needs.Deploy-Dataspace.outputs.registration_service_url }}
```

Comment:
As we are not upgrading registration service dependency in this PR, so this change is not needed, correct ?

Reply:
as I understand this change is only a refactoring because registration_service_url contains now the path prefix, so effectively nothing changes in this line
<img width="1396" alt="Screenshot 2022-07-29 at 09 16 50" src="https://user-images.githubusercontent.com/4217554/181704431-70e5a618-d5c5-423e-a333-b5492054aec8.png">

Reply:
Thanks @Izzzu